python/ql/test/3/library-tests/PointsTo/typehints/test.py

```diff
@@ -8,11 +8,3 @@ def bar(s:set)->Set:
 
 t1 = Optional[Optional[int]]
 t2 = Optional[int][int]
-
-# ODASA-8075
-# Commented out until the fix has been pushed to LGTM.com
-#class baz():
-#    pass
-#
-#while True:
-#    baz = baz[baz]
```

Comment:
I'd hope the fix has been pushed to LGTM.com by now, so perhaps we should uncomment the code again.
```suggestion
class baz():
    pass

while True:
    baz = baz[baz]
```

Reply:
👍  I've put back the test and updated the test results.

Reply:
Oh yeah, I fixed the underlying problem ages ago. Good catch, @aibaars. 🙂